docs: Adding information about the openvsx rate limits and usage tiers (#3018)
codenvy/che-docs: modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc

----
<1> For example: `openVSXURL: "pass:c,a,q[https://open-vsx.org]"`.
+
[WARNING]
====

> To ensure the stability and performance of the community-supported Open VSX Registry, API usage is organized into defined tiers. These limits are implemented by the Eclipse Foundation to protect infrastructure from high-frequency automated traffic and to provide consistent service quality for all users. More details about the link:https://github.com/EclipseFdn/open-vsx.org/wiki/rate-limiting[Rate Limits and Usage Tiers] can be found in the link:https://github.com/EclipseFdn/open-vsx.org/wiki[open-vsx.org wiki].

Reviewer: ```suggestion
To ensure the stability and performance of the community-supported Open VSX Registry, API usage is organized into defined tiers. The Eclipse Foundation implements these limits to protect infrastructure from high-frequency automated traffic and to provide consistent service quality for all users. For more information, see link:https://github.com/EclipseFdn/open-vsx.org/wiki/rate-limiting[Rate Limits and Usage Tiers] and the link:https://github.com/EclipseFdn/open-vsx.org/wiki[open-vsx.org wiki].
```
